swagger/public_v1/template.yml

```diff
@@ -387,3 +375,30 @@ components:
             $ref: "#/components/schemas/SubjectResource"
         jsonapi:
           $ref: "#/components/schemas/JSONAPI"
+    StandardErrorResponse:
+      description: "This schema is used to return a collection of non resourceful application errors."
```

Comment:
What's a `non resourceful application errors`?

Reply:
It's referring to model based resources. I guess the language requires a little tweak!